compute_endpoint/tests/unit/test_endpointmanager_unit.py

```diff
@@ -190,8 +193,18 @@ def test_sets_process_title(
 
 
 @responses.activate
-@pytest.mark.parametrize("status_code", [409, 423, 418])
-def test_gracefully_exits_if_in_conflict_or_locked(
+@pytest.mark.parametrize(
+    "status_code",
+    [
+        HTTPStatus.BAD_REQUEST,
+        HTTPStatus.NOT_FOUND,
+        HTTPStatus.CONFLICT,
+        HTTPStatus.UNPROCESSABLE_ENTITY,
+        HTTPStatus.LOCKED,
+        HTTPStatus.IM_A_TEAPOT,
+    ],
+)
+def test_gracefully_exits_if_registration_blocked(
```

Comment:
Similar suggestion as with the other test (perhaps obviously, since the code under tests is ... similar).